TheWarWithin/Priorities/RogueAssassination.simc

```diff
@@ -43,30 +45,24 @@ actions.cds+=/variable,name=deathmark_kingsbane_condition,value=!talent.kingsban
 actions.cds+=/variable,name=deathmark_condition,value=!stealthed.rogue&buff.slice_and_dice.remains>5&dot.rupture.ticking&buff.envenom.up&!debuff.deathmark.up&variable.deathmark_ma_condition&variable.deathmark_kingsbane_condition
 # Usages for various special-case Trinkets and other Cantrips if applicable
 actions.cds+=/call_action_list,name=items
-## Invoke Externals to Deathmark
-## actions.cds+=/invoke_external_buff,name=power_infusion,if=dot.deathmark.ticking
```

Comment:
I usually keep these commented out for reference.

Reply:
roger